apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -1143,18 +1143,14 @@ export const PoolPoster = ({
                       {intl.formatMessage(
                         {
                           defaultMessage:
-                            "Please contact the <a>{name} team</a> by email if you have <strong>any questions</strong> or <strong>require an accommodation</strong> during this process.",
-                          id: "rKUVdL",
+                            "Please <a>contact the team</a> by email if you have <strong>any questions</strong> or <strong>require an accommodation</strong> during this process.",
+                          id: "YK/RNP",
                           description:
                             "Opening sentence asking if accommodations are needed",
                         },
                         {
-                          a: (chunks) => anchorTag(chunks, contactEmail),
-                          name: () => (
-                            <>
-                              {getLocalizedName(pool.team?.displayName, intl)}
-                            </>
-                          ),
+                          a: (chunks: ReactNode) =>
+                            anchorTag(chunks, contactEmail),
```

Comment:
For some reason, Typescript only started complaining after the `name: ...` was removed